src/pages/about.js

```diff
@@ -48,9 +49,52 @@ module.exports = React.createClass({
         </span>
       );
     }
+
     this.setState({
       aboutCopy: aboutCopy
     });
+
+    var percentIncrement = 10; // We'll track the video progress in 10% increments.
+    if (this.props.test === "video") {
+
+      this.setState({
+        nextPercentDone: percentIncrement,
+        percentIncrement : percentIncrement,
+        videoStarted : false
+      });
+
+      this.refs.videoPlayer.addEventListener("play", this.onVideoStart);
```

Comment:
you have to use `e => this.onVideoStart(e)` here otherwise you'll lose `this` inside the function (same for the line below).

Reply:
Happy to change, but.. it works!

Reply:
react might be auto-binding some of these functions for you but I wouldn't risk it...any update to react will break this

Reply:
For sure, will update!